.travis.yml

```diff
@@ -33,17 +69,68 @@ before_script:
   - misc/install-perl-module.pl JSON
   - misc/install-perl-module.pl Path::Tiny
   - misc/install-perl-module.pl Test::Exception
+  # install libuv >= 1.0.0 (optionally required for building / testing libh2o)
+  - |
+    case "${TRAVIS_OS_NAME}" in
+      "linux" )
+        LIBUV_VERSION=1.0.0
+        if [ "`cat ${CACHE_VERSION_DIR}/libuv 2> /dev/null`" != "${LIBUV_VERSION}" ] ; then
+          curl -L https://github.com/libuv/libuv/archive/v${LIBUV_VERSION}.tar.gz | tar xzf -
+          (cd libuv-${LIBUV_VERSION} && ./autogen.sh && ./configure --prefix=${INSTALL_DIR} && make && make install)
+          echo "${LIBUV_VERSION}" > ${CACHE_VERSION_DIR}/libuv
```

Comment:
You should not create a version file, unless the installation steps succeed (and this might also apply for the case blocks in other parts of the PR).